.github/workflows/acceptance-tests.yml

```diff
@@ -14,17 +14,13 @@ jobs:
   acceptance-tests:
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [14.x]
-
     steps:
       - run: echo Running on target url $ACCEPTANCE_URL
       - uses: actions/checkout@v2.3.4
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Use Node.js 16
```

Comment:
nit: you can remove these names, they're fairly redundant and what the action is doing is quite clear from the name itself